cucumber-android/src/main/java/cucumber/api/android/CucumberAndroidJUnitRunner.java

```diff
@@ -0,0 +1,58 @@
+package cucumber.api.android;
+
+import android.os.Bundle;
+import android.support.test.runner.AndroidJUnitRunner;
+
+import java.io.File;
+
+import cucumber.api.CucumberOptions;
+import cucumber.runtime.android.CucumberJUnitRunnerBuilder;
+
+/**
+ * Android Orchestrator compatible replacement for {@link cucumber.api.android.CucumberInstrumentation}.
+ */
+public class CucumberAndroidJUnitRunner extends AndroidJUnitRunner {
+
+    private static final String ARGUMENT_ORCHESTRATOR_RUNNER_BUILDER = "runnerBuilder";
+
+    private Bundle arguments;
+
+    @Override
+    public void onCreate(final Bundle bundle) {
+        bundle.putString("plugin", getPluginConfigurationString()); // we programmatically create the plugin configuration
+        bundle.putString(ARGUMENT_ORCHESTRATOR_RUNNER_BUILDER, CucumberJUnitRunnerBuilder.class.getName());
```

Comment:
we need to pass `CucumberJUnitRunnerBuilder` to `AndroidJUnitRunner` so then it will automatically be asked to create `CucumberJUnitRunner`